src/main/resources/search_suggestions/countries.txt

```diff
@@ -0,0 +1,196 @@
+Afghanistan
```

Comment:
This entire file must be parsed and countries must be validated to share the same name as present on the json files that countries are reference (`countriesHD.json`) EX. "United States" in this file must be changed to "United States of America" as presented in the json.